src/compiler/utilities.ts

```diff
@@ -4641,4 +4641,27 @@ namespace ts {
     export function unescapeIdentifier(identifier: string): string {
         return identifier.length >= 3 && identifier.charCodeAt(0) === CharacterCodes._ && identifier.charCodeAt(1) === CharacterCodes._ && identifier.charCodeAt(2) === CharacterCodes._ ? identifier.substr(1) : identifier;
     }
+
+    export function levenshtein(s1: string, s2: string): number {

```

Comment:
Did you intend for `levenshtein` to be in the public TypeScript API? This file has two top-level `ts` namespaces, the first marked with `/* @internal */` and the second without the marker. This code is in the second namespace currently.